arm/variants/max3263x_fthr/variant.h

```diff
@@ -79,16 +80,25 @@
 #define PIN_LED     RED_LED
 #define LED_BUILTIN RED_LED
 
+// Push button
+// ----
+#define SW1 P2_3
+#define SW2 NOT_CONNECTED
+#define SW3 NOT_CONNECTED
+// Standardized button names
+#define BUTTON1 SW1
+
```

Comment:
Are these required?

Reply:
I think defining SW1 and BUTTON1 is useful as people may use those in their original mBed code as these names are shown on the pinout diagram of this board.
As for SW2 and SW3 i included them as they are there in the pindefinition for mBed of this board, but as these pins of the µCU are not lead out on the board, people will not use these names in their code, we can just drop them.